src/Analyzers/Core/Analyzers/SimplifyInterpolation/Helpers.cs

```diff
@@ -14,9 +14,14 @@
 
 namespace Microsoft.CodeAnalysis.SimplifyInterpolation
 {
-    internal static class Helpers
+    internal class Helpers
     {
-        public static void UnwrapInterpolation<TInterpolationSyntax, TExpressionSyntax>(
+        protected virtual SyntaxNode GetPreservedInvocationInstanceSyntax(IInvocationOperation invocationOperation)
+        {
+            return invocationOperation.Instance.Syntax;
+        }
+
+        public void UnwrapInterpolation<TInterpolationSyntax, TExpressionSyntax>(
```

Comment:
alternative approach.  update this helper to have TConditional/TParenthesized.  this helper can then check the syntax shape and adjust right here.

Reply:
This is a very C#-specific thing to be checking. What would the parameters be for VB?

Reply:
Just checked, VB never removes parens anyway, so there is no downside to hardcoding this across all languages.
https://sharplab.io/#v2:EYLgtghgzgLgpgJwDQBsQDdhJhxBLAMwE8AfASTAAcB7BGKAAgGUjY4wBYAKAAUBXYCjwBjBgGEU0RmO4M5DfoJEMAYnwB2wmHmrqGAWQAUASgYBBRkxgI86gOaz5TgEpwYfBHoAkAIgDehgBMxgB0ACrUVjb2JgC+Po5yAKLqACaqGlo66twp6RJSQA

Reply:
Implemented.